app/assets/javascripts/components/stories/StoryLink.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import Icon from 'components/shared/Icon';
+import hoverTemplate from 'templates/story_hover.ejs';
+import noteTemplate from 'templates/note.ejs';
+
+const STATE_ICONS = { accepted: 'check_circle', rejected: 'cancel' };
+
+export default class StoryLink extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick() {
+    const { story } = this.props;
+    document.getElementById(`story-${story.get('id')}`).scrollIntoView();
+    story && _.each(story.views, view => view.highlight());
+  }
+
+  renderIcon(state, className = '') {
```

Comment:
Take care when using the name `state` inside a React component, it can be confusing. Is it the state of the story? If so, what do you think about `storyState`?